packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/explore/TablesViewer.tsx

```diff
@@ -21,22 +19,18 @@ import { Table, TableBody, TableCell, TableHead, TableHeader, TableRow } from ".
 import { bufferToBigInt } from "../utils/bufferToBigInt";
 import { EditableTableCell } from "./EditableTableCell";
 
-type Props = {
-  table: string | undefined;
-};
+const emptyColumnSort: SortingState = [];
```

Comment:
could also call this `initialSortingState`, which I use as a pattern quite often and might be clearer than either approach we had before

Reply:
agreed, a better name, renamed